q2-structure/proposal-by-xdg.md

```diff
@@ -0,0 +1,219 @@
+
+## Voting
+
+### Vote Administration
+
+Every vote requires a Vote Administrator who manages communication, collection
+of secret ballots, and all other necessary activities to complete the voting
+process.
+
+Unless otherwise specified, the Steering Council selects the Vote
+Administrator.
+
+A Vote Administrator must not be a member of the Steering Council nor a
+candidate or subject of the vote.  A Vote Administrator may be a member of the
+Core Team and, if so, may cast a vote while also serving as administrator.  If
+the Vote Administrator becomes a candidate during an election vote, they will
+appoint a non-candidate replacement.
+
+If the entire Steering Council is vacant or is the subject of a No Confidence
+Vote, then the Core Team will select a Vote Administrator by consensus.  If
+consensus cannot be reached within one week, the President of The Perl
+Foundation will select a Vote Administrator.
+
+### Term and Special Elections
+
+Term and Special Election are ranked-choice votes to construct an ordered list
+of candidates to fill vacancies in the Steering Council.
+
+Both Term and Special Elections occur in two stages:
+
+1. Candidates advertise their interest in serving. Candidates must be
+   nominated by a Core Team member. Self-nominations are allowed.  Nominated
+   candidates may share a statement about their candidacy with the Core Team.
+
+2. Active Core Team Members vote by ranking all candidates.  Voting is
+   performed anonymously.  After voting is complete, candidates are ranked
+   using the Condorcet Minimax method.  If a tie occurs, it may be resolved by
+   mutual agreement among the tied candidates, or else the tie will be
+   resolved through random selection by the Vote Administrator.
+
+For a Term Election, each phase will last two weeks.  At the end of the second
+phase, the top five ranked candidates are elected as the new Steering Council.
+
+For a Special Election, each phase will last one week.  At the end of the
+second phase, vacancies are filled from the ordered list of candidates until
+no vacancies remain.
+
+If a Steering Council candidate is not already a member of the Core Team, they
+are added to the Core Team if elected to the council.
+
+### Membership Change, Amendment, and No Confidence Votes
+
+Membership Change, Amendment, and No Confidence Votes are super-majority votes
+by the Core Team.
+
+They occur in two steps:
+
+1. The Vote Administrator describes proposal being voted upon.
+
+2. Active Core Team members vote in favor or against the proposal.  Voting is
+   performed anonymously.
+
+If 2/3 or more of the votes cast are in favor of the proposal, the proposal
+succeeds and goes into effect.
+
+For a Membership Change Vote, phase 2 will last one week.  For Amendment and
+No Confidence Votes, phase 2 will last two weeks.
+
+---
+
+<a id="note1" href="#note1ref"><sup>1</sup></a> Given vX.Y.Z, a feature
+release is a bump in the first or second version number.
```

Comment:
The wording above was "stable feature release", I think the wording should agree with this note.

Reply:
I left it out because we bump Y for dev releases.  But I'll clarify "stable" in an additional sentence.